app/controllers/api/physical_servers_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  class PhysicalServersController < BaseController
+    def show
+      if params[:c_id]
+        physical_server = PhysicalServer.find(params[:c_id])
+        response_payload = physical_server.as_json
+        response_payload['host'] = case physical_server.host
+                                   when nil then nil
+                                   else physical_server.host.id
+                                   end
```

Comment:
@chessbyte is better `response_payload['host']` or `response_payload['host_id']`? Once we will need the ID of the host to perform other request in host resource.